lalrpop/src/main.rs

```diff
@@ -179,24 +179,24 @@ mod test {
         vals.iter().map(|v| v.into()).collect()
     }
 
-    fn parse_args_vec(args: &Vec<&str>) -> Args {
+    fn parse_args_vec(args: &[&str]) -> Args {
```

Comment:
```suggestion
    fn parse_args(args: &[&str]) -> Args {
```
It's not  a vec anymore